osc-service-api/src/main/java/org/osc/core/broker/service/dto/VMPortDto.java

```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.core.broker.service.dto;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import io.swagger.annotations.ApiModelProperty;
+
+@XmlRootElement(name = "vmPort")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class VMPortDto extends BaseDto {
```

Comment:
Dont call it VMPort but just Port.  This is a little more generic and can also represent a port of a device that is not a VM, i.e.: router, container, etc. Also rename all the methods, paramaters, variables to reflect this, i.e.: setVMPorts. @arvindn05  do you agree?